app/src/main/java/org/fossasia/openevent/general/event/EventsViewModel.kt

```diff
@@ -24,6 +24,7 @@ class EventsViewModel(private val eventService: EventService, private val prefer
     val error: LiveData<String> = mutableError
     private val mutableShowShimmerEvents = MutableLiveData<Boolean>()
     val showShimmerEvents: LiveData<Boolean> = mutableShowShimmerEvents
+    private val mutableLastSearch = MutableLiveData<String>()
```

Comment:
Why mutable String? Can't you use a simple String instead?

Reply:
You are right, it still works fine with String so I'm sending a new commit now